js/constants/appConfig.js

```diff
@@ -36,8 +36,8 @@ module.exports = {
   cookieblock: {
     enabled: true
   },
-  autoplay: {
-    enabled: false
+  noAutoplay: {
+    enabled: true
```

Comment:
If the default is false, the notification won't popup in most of the cases because https://github.com/brave/browser-laptop/pull/8751/files#diff-d4f09c262fdb2f86d8f7407afb926c8eR21
The only case it will show up is users toggle global autoplay to block and without site setting of autoplay to the website

The setting is similar to our default `fullscreen` setting.